### DISCUSSION TITLE
WIKI ändern

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1917] [Q&A]
Derzeit sieht es so aus, scheint einige zu verwirren. Einige denken anscheinend, es gäbe keine Doku für v3?
https://forum.iobroker.net/topic/49776/jarvis-v3-0-0-just-another-remarkable-vis/3019
![image](https://user-images.githubusercontent.com/84818934/192382511-5686a8d0-59c7-4581-ac35-d9171fbc8197.png)
Vielleicht sollte unter 
v2 

Community: @MCU  https://mcuiobroker.gitbook.io/jarvis-infos/ 

und v3 der Eintrag 

Community: @MCU  https://mcuiobroker.gitbook.io/jarvis-infos/ ?

**Comment by Zefau** [comment]
Ich hab dich mal ins Repository eingeladen. Hast du damit Berechtigungen, dass du das ändern kannst?

Gestalte es gerne frei nach deinen Vorstellungen.

Vielen Dank für deine Unterstützung.

**Comment by mcu** [comment]
Benutze ich schon lange nicht mehr ioBroker.

"Hast du damit Berechtigungen, dass du das ändern kannst?" - Ich weiß
nicht, wie ich es ändern soll.

Ich habe einen Übersetzer benutzt, wenn etwas nicht stimmt, tut es mir leid

**Reply by mcuiobroker** [reply]
@Zefau Nimm die Einladung raus. Scheint zu Verwirrung zu kommen. Warum auch immer.?
Änderung passt auch.

**Reply by mcuiobroker** [reply]
@zefau Du hast mcu falsch verknüpft.

**Reply by Zefau** [reply]
![image](https://user-images.githubusercontent.com/14368811/192853262-c66d618f-a41d-45a4-9e83-78829b4543cf.png)

Hab es rausgenommen und neu hinzugefügt, aber sieht aus wie vorher.
